setup.cfg

```diff
@@ -34,6 +34,7 @@ install_requires =
     openai<=1.1.0
     python-multipart<=0.0.9
 python_requires = >=3.8
+    ruff==0.6.7
```

Comment:
Please move this above python requires and use ruff<=0.6.7

Reply:
Also please update python_requires to 3.9

Reply:
Yes sure will do